src/Hotel.java

```diff
@@ -5,24 +5,29 @@
     private String name;
     private ArrayList<Room> rooms;
 
-    Hotel(String name, ArrayList<Room> rooms) {
+    Hotel(String name, ArrayList<Room> rooms)
+    {
         this.name = name;
         this.rooms = rooms;
     }
 
-    String getName(){
+    String getName()
+    {
         return this.name;
     }
 
-    ArrayList<Room> getRooms(){
+    ArrayList<Room> getRooms()
+    {
         return this.rooms;
     }
 
-    int hotelNumberOfRooms(){
+    int hotelNumberOfRooms()
+    {
         return this.rooms.size();
     }
 
-    ArrayList<Room> findAllAvailableRoomsForIntervalAndSize(Date from, Date to, int numberOfPeople) {
+    ArrayList<Room> findAllAvailableRoomsForIntervalAndSize(Date from, Date to, int numberOfPeople)
```

Comment:
Also when you use an object parameter in a method, check if the objects are null, before using them.